### PR TITLE
Adding History of the Standardization

### LINKED ISSUE #18: History of the standardization is unknown
Hello WoT Marketing Task Force,

I am submitting this issue for a **feature request** with following details.

**Problems:**

* There is no clear and concrete information on the standardization's history, which makes it time-consuming to prepare such information to present to newcomers.
* History is understandable only through blog posts in the IG website.

**Requirements:**

* A chronological history of the WoT Standardization activity in W3C.
* A list of important events in the WoT Standardization activity in W3C.

**Solution Proposals:**

* A markdown file that can read on GitHub or desktop that lists the events in a chronological order. Important events would be then marked with bold or a special symbol.
* This can be then copied into presentations, Wikipedia, our wiki.

**Background information:**

This issue has been created when the work on Wikipedia has started. There it was realized that it is difficult to convey how everything has all started. In the meantime, I consider it powerful when we can show how much work has been put into W3C WoT by giving a timeline.

### PR DESCRIPTION
**Pull request by egekorkan**
fixes #18 

I have also rearrange the contact into about that also contains the history. I have checked the statistics and our contact page got 17 clicks out of 4000+ total as the first entry page (someone knows this link directly) and 59 out of 7000+ as a visit. Thus, I think that we do not have much of a risk with changing its link or location. We can also keep the old link but move to this new location. Any opinions?

### CODE REVIEW

**Comment by egekorkan** [comment]
Call of 09.08:
- We can put photos for each event if we find them. Worst case, some more generic pictures about a plugfest we did
-

**Review by danielpeintner — APPROVED** [review]
Content-wise it looks good.

Let's add some images to make it look better..

**Comment by egekorkan** [comment]
Main Call of 10.08:
- we should add the second w3c wot workshop at munich

**Comment by egekorkan** [comment]
I have added some pics but not happy with how it looks at the moment. I will iterate on it later on.

**Comment by egekorkan** [comment]
Also, I have just realized that netlify previews do not contain the images.

**Comment by egekorkan** [comment]
I have decided to comment out the images for now. It would be better to have more to choose from since I could not find many, even for the more recent events. @ashimura I know that you upload the photos to wiki but is there way to see all of them?

**Comment by egekorkan** [comment]
@mmccool will review the text and after TPAC we can have a look with more photos.

File: docs/about/history/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		<img src="/WoT/images/ws2019-Group-photo.png" width="100%" alt="Second WoT Workshop">
+		<figcaption style="text-align: center;">Second W3C WoT Workshop in Munich, Germany</figcaption>
+	</figure> -->
+
+	<br>
+	
+	<p>
+	Since then, the Working Group is working on new specifications such as the WoT Profiles and WoT Discovery, while
+	working on the newer versions of Thing Description, Architecture, Scripting API and Binding Templates.
+	If you are interested in learning about WoT, check our <a href="/WoT/documentation"> documentation page</a>.
```
Comment (danielpeintner) [inline docs/about/history/index.html:82]: I think this `href` should read differently and without "WoT"

`href="{{'documentation/' | relative_url }}"`

Comment (egekorkan) [reply docs/about/history/index.html:82]: Could you test it on your local machine? The behavior of links in netlify is different

Comment (danielpeintner) [reply docs/about/history/index.html:82]: Your code works locally but fails on netlify (I guess it fails also on the website in the end).

Moreover, my proposed code is how we do it elsewhere and works locally.. also.. and I am pretty sure it works also on netlify... 

Another note: Adding "WoT" prefix to each subpage would not be a good idea if this prefix changes once... we deal with it in `_config.yml`.

Hope this helps.

Comment (egekorkan) [reply docs/about/history/index.html]: Done the first part. Not sure what you meant by:
`Another note: Adding "WoT" prefix to each subpage would not be a good idea if this prefix changes once... we deal with it in _config.yml.` We can handle that as a separate issue

**Pull request merged**